### PR TITLE
Centos 8 support

### PR DESCRIPTION
**Pull request by TheREK3R**

### CODE REVIEW

**Comment by TheREK3R** [comment]
[yum-osuosl](https://trello.com/c/FqsmXRLR/1849-yum-osuosl)

File: spec/spec_helper.rb [hunk 1]
```diff
@@ -1,6 +1,11 @@
 require 'chefspec'
 require 'chefspec/berkshelf'
 
+CENTOS_8 = {
+  platform: 'centos',
+  version: '8',
+}.freeze
+
```
Comment (ramereth) [inline spec/spec_helper.rb:8]: Please add this to ``ALL_PLATFORMS`` below.

**Comment by TheREK3R** [comment]
!bump patch

**Comment by osuosl-manatee** [comment]
Jenkins has merged this PR into `master` and has automatically performed a patch-level version bump to v1.1.1.  Have a nice day!

**Pull request merged**